ui/app/rif/pages/rns/subdomains/index.js

```diff
@@ -74,7 +116,7 @@ class Subdomains extends Component {
           </button>
         </div>
         <div className="list">
-          {body}
+          {list}
```

Comment:
can we rename subDomainList? Or lstSubDomain?

Reply:
I could but the component itself is subdomains so i guess is more than explicit right?